app/views/base/_file_manager_extra_tools.html.erb

```diff
@@ -1,7 +1,18 @@
-<div class="col-md-12" id="uploading-stuff">
-  <%= render "server_upload" if controller.respond_to?(:server_upload) %>
-</div>
-<div class="col-md-12" id="local-upload">
-  <%= render "local_upload" if @change_set.respond_to?(:files) %>
+<div class="col-md-12" id="upload-tools">
+  <a class="btn btn-primary" data-toggle="collapse" href="#uploading-stuff">Ingest from Server</a>
```

Comment:
I'm not sure what "uploading-stuff" means 😂 

Oh this is what we call the collapsable content area? What about "server-ingest-pane" and "local-ingest-pane"?